project/Dependencies.scala

```diff
@@ -12,7 +12,7 @@ import sbt._
 object Dependencies {
 
   object Versions {
-    val crossScala = Seq("2.12.4", "2.11.8")
+    val crossScala = Seq("2.12.6", "2.11.8")
```

Comment:
Can we up the Scala version to 2.12.11?

Reply:
we can however i intentionally avoided this
2.12.11 introduces some changes that break dex
My plan was to have the same version across all our projects for now and then create a seperate story to update all projects to 2.12.11

Reply:
Sounds like a plan